libs/ui-toolkit/pipes/src/lib/keep-html.pipe.ts

```diff
@@ -0,0 +1,13 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { DomSanitizer } from '@angular/platform-browser';
+/**
+ * Pipe for HTML sanitizer.
+ */
+@Pipe({ name: 'keepHtml' })
+export class EscapeHtmlPipe implements PipeTransform {
```

Comment:
EscapeHtmlPipe is diffenrent from KeepHtmlPipe. Also, I guess you miss the gor prefix. Please, fix the both things